### DISCUSSION TITLE
Welcome to dashboard-v2 Discussions!

### DISCUSSION THREAD

**Discussion by ghost** [RealToken-Community/dashboard-v2#38] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community for work on Dashboard!
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Kurtisone** [comment]
Can we talk in French too ? Or another place will be create for ?

**Comment by daodevinc** [comment]
Gm here !

**Comment by NandyBa** [comment]
Hi community members ! ⚡️
